src/odemis/driver/test/lakeshore_test.py

```diff
@@ -35,36 +35,82 @@
 TEST_NOHW = (os.environ.get("TEST_NOHW", "0") != "0")  # Default to Hw testing
 
 # arguments used for the creation of basic components
-CONFIG = {"name": "Lakeshore Test",
-          "role": "temperature-controller",
-          "port": "/dev/ttyUSB0",
-          "sensor_input": "B",
-          "output_channel": 2,
+CONFIG_SERIAL_BUS_CONNECTION = {
+    "name": "Lakeshore Test",
+    "role": "temperature-controller",
+    "port": "/dev/ttyUSB0/",
+    # "port": "/dev/fake",
+    "sensor_input": "B",
+    "output_channel": 2,
+}
+
+# arguments used for the creation of basic components
+CONFIG_IP_BUS_CONNECTION = {
+    "name": "Lakeshore Test",
+    "role": "temperature-controller",
+    "port": "192.168.30.214",
+    # "port": "/dev/fake",
+    "sensor_input": {"monolith": "A", "sample": "B"},
+    "output_channel": {"monolith": 1, "sample": 2},
 }
 
 # arguments used for the creation of basic components
 if TEST_NOHW:
-    CONFIG["port"] = "/dev/fake"
+    CONFIG_SERIAL_BUS_CONNECTION["port"] = "/dev/fake"
+    CONFIG_IP_BUS_CONNECTION["port"] = "/dev/fake"
+
+
+class LakeshoreBaseTest:
+    """Base class for testing different models of LakeShore temperature controller."""
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.dev.terminate()  # free up socket.
+
+    def test_temperature_poll(self):
+        return
+        # Test that the temperature polls.
+        # Designed to work with the simulator - the temperature will not be exactly the same
+        logging.debug("Test stasis temperature polling")
+        temp1 = self.dev.temperature.value
+        time.sleep(3)
+        temp2 = self.dev.temperature.value
+        # self.assertNotEqual(temp1, temp2)
```

Comment:
I think we should uncomment because otherwise it doesn't test anything.
However, I can imagine it's not great, as it might randomly return the same value. So 10% of the time the test case will fail. Maybe instead, it could be a loop of 10, every second, and check that not all the readings are the same.